backend/app/services/config_service.py

```
@@ -0,0 +1,135 @@
+import json
+import sqlite3
+from pathlib import Path
```

Ruff (0.12.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)